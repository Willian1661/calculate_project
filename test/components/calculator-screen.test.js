const calculatorScreen = require("../../src/components/calculator-screen")

describe("calculator screen component", () =>{

    it("is passing calculator screen", () => {
        expect(calculatorScreen("baz","foo")).toMatchSnapshot()
    });

})