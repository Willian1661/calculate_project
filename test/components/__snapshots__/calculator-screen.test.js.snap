// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator screen component is passing calculator screen 1`] = `
"
    <div id="baz">
        <input type="text" placeholder="0" id="foo">
    </div>
    "
`;
